modules/logging-loki-storage-aws.adoc

```diff
@@ -25,3 +25,22 @@ $ oc create secret generic logging-loki-aws \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
 ----
+
+== AWS storage for STS enabled clusters
```

Comment:
I think we also specify the object storage secret to be configured on LokiStack CR on AWS clusters when using CCO since we don't need to specify access key_id and access_key_secret for CCO flow.

The object storage to be used for Loki on AWS using CCO should be 
oc -n openshift-logging create secret generic "logging-loki-s3" \
  --from-literal=bucketnames="<s3_bucket_name>" \
  --from-literal=region="<bucket_region>"

Reply:
@libander This secret is not related to the secret created by CCO. It is the object storage secret that is created to provision the LokiStack CR like on [here](https://github.com/openshift/openshift-docs/pull/73846/files#diff-74ccd0ed6e0b9fad40a29eff160552ffb44090f629d79d6f1c823c3f89a4eb67L17)